src/components/dropdown-disclosure/components/list-item/index.tsx

```diff
@@ -122,6 +122,31 @@ const DropdownDisclosureLinkItem = ({
 	)
 }
 
+/*
+ * This component provides a temporary workaround using an anchor tag
+ * instead of 'link' component. There has been an issue observed where
+ * 404 links throw uncaught error pages with the version switcher drpodown
+ */
+
+const DropdownDisclosureAnchorItem = ({
```

Comment:
Created a new component here so that the user dropdown component will still use `Link` and the anchor tag use is contained to just the version switcher dropdown.